docs/services/slam/cartographer/_index.md

```diff
@@ -343,9 +440,8 @@ See [Attributes](#attributes) for details.
 }
 ```
 
-{{% /expand%}}
-
-#### Attributes
+{{% /tab %}}
+{{< /tabs >}}
 
 <!-- prettier-ignore -->
```

Comment:
The attributes table seems a little out of place here (in the middle of the "Localize only" section) - should we move it after "Localize only" and give it a header?

Reply:
let me try putting them with the configuration sections